react/modules/imageAsList.tsx

```diff
@@ -27,7 +28,9 @@ export const getImagesAsJSXList = (
         alt={description}
         maxHeight={height}
         width={width}
-        experimentalPreventLayoutShift={experimentalPreventLayoutShift}
+        experimentalPreventLayoutShift={
+          experimentalPreventLayoutShift ?? experimentalPreventLayoutShiftChild
```

Comment:
Shouldn't the child have precedence over the list?

Reply:
Good point 🤔

Reply:
Honestly though I can't think of a situation where you'd want a specific image to behave differently than the others in the list. The intent was for the list prop to function as an override for the individual images